protocol/proto/metadata.proto

```diff
@@ -50,6 +50,7 @@ message Album {
         ALBUM = 0x1;
         SINGLE = 0x2;
         COMPILATION = 0x3;
+	EP = 0x4;
```

Comment:
Indentation is wrong here

Reply:
Sorry I did miss that the file uses space indentation everywhere